CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 A curated, chronologically ordered list of notable changes in [Gitpod's default workspace images](https://hub.docker.com/u/gitpod).
 
+## 2023-10-18
+
+- Node 20 replaces Node 18 as the current LTS
+- Introduce Node `21.0.0` in `workspace-node`
+- Introduce `workspace-node-18` for those, who aren't ready to upgrade just yet
```

Comment:
This is a good idea, having a separate node 18 image, because it'll get security updates [till (30 Apr 2025)](https://endoflife.date/nodejs)

Reply:
I will document our strategy of supporting language versions until they stop receiving security maintenance in a follow up.